### ISSUE TITLE
Add support for UTF-8 in message

### ISSUE THREAD

**Issue by rbosse** [mvz/email-outlook-message-perl#14]
Had this error:
Wide character in print at /usr/bin/msgconvert line 58.

**Comment by sigurdtheone** [comment]
This can be solved by adding "use open ":std", ":encoding(UTF-8)";" to the top of /home/<your user>/perl5/bin/msgconvert

**Comment by mvz** [comment]
If anyone seeing this error can provide the following, that would be great:
- A msg file that triggers the problem
- The command line you use to trigger the problem

**Comment by ojwb** [comment]
There's actually a file which triggers this in the testsuite already, and no other command line options are required:

```
$ script/msgconvert t/files/plain_uc_wc_unsent.msg
Wide character in print at script/msgconvert line 58.
```

I'm using Debian unstable.

(I'm not yet sure if this is the same problem or not, but FWIW I'm getting a similar issue in my own code when I request `body_str` on an Email::MIME object I get from Email::Outlook::Message:

```
Wide character at /usr/share/perl5/Email/MIME.pm line 376.
```

I'm in the process of investigating, went to check if someone had reported it already, and found this ticket and that the file I was testing with triggers this issue too.)

**Comment by ojwb** [comment]
@mvz I found that the output file in msgconvert used to be set to be `:utf8` but that was removed by #7.

In #7 it is mentioned that things are still broken until https://github.com/rjbs/Email-MIME/pull/36 is merged in Email::MIME, but that PR doesn't seem to have gone well and was closed without merging.

It's still unclear to me exactly what's broken and where, but I don't think it's msgconvert (or at least not entirely) as I'm seeing similar issues with my own code.
